docs/mine/lotus/split-markets-miners.md

```diff
@@ -68,51 +104,66 @@ export LOTUS_BACKUP_BASE_PATH=/tmp
+To initialize the markets service we need to create a seed `config.toml` for
+the markets node. For more information see the [configuration page](https://docs.filecoin.io/get-started/lotus/configuration-and-advanced-usage/) and the [custom storage layout page](https://docs.filecoin.io/mine/lotus/custom-storage-layout/).
 
-In the example commands below, we have placed the `config.toml` in the `/tmp` directory. The repository directory which we use is `~/.lotusmarkets`. We are using the following `config.toml`:
+In the examples below the `config.toml` is in the `/tmp` directory.
+The repository directory is `~/.lotusmarkets`.
+We are using the following `config.toml`:
 
 ```toml
 [API]
+  # Endpoint for API calls to the markets process
   ListenAddress = "/ip4/127.0.0.1/tcp/8787/http"
   RemoteListenAddress = "127.0.0.1:8787"
 
 [Libp2p]
+  # Endpoint for libp2p requests (public)
   ListenAddresses = ["/ip4/0.0.0.0/tcp/24001", "/ip6/::/tcp/24001"]
   AnnounceAddresses = ["/ip4/12.34.56.78/tcp/24001"]
 ```
 
-If you intend to run the `mining/sealing/proving` node on the same machine as the `markets` node, make sure that their listeners do not clash. By default the `lotus-miner` API server listens to port 2345, so in the example configuration above, we change the configuration for the `markets` node to listen to `127.0.0.1:8787`.
+If you intend to run the `mining/sealing/proving` node on the same machine as the
+`markets` node, make sure that their listeners do not clash.
+By default the `lotus-miner` API server listens to port 2345, so in the example
+configuration above, we change the configuration for the `markets` node API server
+to listen to `127.0.0.1:8787`.
 
-Make sure you adjust the `[Libp2p]` section on the `markets` node accordingly - it needs to be publicly accessible, so that clients can make storage and retrieval deals with your system.
+Make sure you adjust the `[Libp2p]` section on the `markets` node accordingly - it
+needs to be publicly accessible so that clients can make storage and retrieval
+deals with your system.
 
-The `[Libp2p]` section on the `mining/sealing/proving` node can be removed because this node will no longer be running a Libp2p node.
+The `[Libp2p]` section of the `config.toml` on the `mining/sealing/proving` node
+can be removed because it will no longer be running a Libp2p node.
 
-### 3. Initialising a `markets` service repository
+#### 3. Initialising a `markets` service repository
 
-1. Create authentication tokens for the `markets` node. This is an online operation, so your `lotus-miner` should be running with its updated configuration (in case you move its API to another IP:PORT)
+1. Create authentication tokens for the `markets` node. This is an online operation,
```

Comment:
If that's the case, there should be an instruction to start the lotus markets process?